git-active-branches

```diff
@@ -1,9 +1,35 @@
 #!/bin/sh
 set -eu
 
-# TODO: Make the max age configurable
+DEFAULT_SINCE='3.weeks.ago'
+since=$DEFAULT_SINCE
+
+usage() {
+    echo "usage: git active-branches [-s|-a <date>]" >&2
+    echo >&2
+    echo "Options:" >&2
+    echo "-s      show branches active since <date> (as in 'git log --since')" >&2
+    echo "-a      (an alias for '-s')" >&2
+    echo "<date>  any date format recognized by 'git log'" >&2
+    echo >&2
+    echo "Unless specified, <date> defaults to \"$DEFAULT_SINCE\". For examples see:" >&2
+    echo "https://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History" >&2
+}
+
+while [[ $# -gt 0 ]]; do
```

Comment:
Let's use the standard test syntax here if we don't need to do anything fancy like prefix or regex matching:

```suggestion
while [ $# -gt 0 ]; do
```

Reply:
Ah, yes, sorry. That's so hard-wired into my muscle memory, that I didn't even notice that slipped in there. It's not even valid Bourne shell syntax, so I'm surprised it didn't throw an error.

I'll fix that real quick in one of these other PRs.